### ISSUE TITLE
Inception-v4 for multi-label prediction

### ISSUE THREAD

**Issue by aldarabs** [tensorflow/models#955]
## Changing Inception-v4 architecture to do Multi-label classification

Hello guys,
I am working on image tagging and annotation problem, simply an image may contain multiple objects. I want to train inception-v4 for multi-label classification. My training data will be an image and a vector of length equals the number of classes and has 1 in each index if the object exists in the image. For example, If I have four classes (Person, car, tree, buildings). If an image contains a person and car. Then my vector will be (1, 1, 0, 0).

What changes do I need to make to train inception-v4 for the tagging and annotation problem?

Do I only need to change the input format and change the loss function from softmax to sigmoid_cross_entropy_with_logits in the inception-v4 architecture?

https://github.com/tensorflow/models/blob/master/slim/nets/inception_v4.py

Thank you in advance.

**Comment by drpngx** [comment]
Hi @aldarabs , great to see you're trying this out. This is probably best asked on stackoverflow, where we monitor all issues with the tag "tensorflow".
